next-cloudinary/src/components/CldOgImage/CldOgImage.tsx

```diff
@@ -56,8 +53,8 @@ const CldOgImage = ({ excludeTags = [], twitterTitle, ...props }: CldOgImageProp
       <meta property="og:image:width" content={`${options.width}`} />
       <meta property="og:image:height" content={`${options.height}`} />
 
-      {options.alt && (
-        <meta property="og:image:alt" content={options.alt} />
+      {alt && (
+        <meta property="og:image:alt" content={alt} />
```

Comment:
Any reason to destructure alt specifically above insetad of referencing it from `options.alt` as you did originally?

Reply:
it was just part of the cleanup to remove it from alt

i dont know that removing it from options matters (per my other comment) but it felt more correct since the options were intended to be for the constructCloudinaryUrl function

Reply:
👍